README.md

```diff
@@ -102,6 +102,25 @@ the secrets stored in temp files and in the Python process environment are gone.
 
     This flag can be useful for when you have secrets that you don't need access to for development. For example API keys for monitoring tools. This flag can be used multiple times.
 
+* `-e, --environment` Specify section (environment) to parse from secret YAML
+
+    This flag specifies which specific environment/section to parse from the secrets YAML file (or string). In addition, it will also enable the usage of a `common` section which will be inherited by other sections/environments. In other words, if your `secrets.yaml` looks something like this:
+
+```yaml
+common:
```

Comment:
Ah, never noticed the `-D` option. That actually makes a lot of sense and inherently enforces the structural organization of environments. It also means you don't need to make a git commit just to add a new environment as you could just add it to S3 and anyone with access could start using it.